web/src/lib/components/shared-components/notification/notification-card.svelte

```diff
@@ -90,7 +90,7 @@
 		</button>
 	</div>
 
-	<p class="whitespace-pre text-sm pl-[28px] pr-[16px]" data-testid="message">
```

Comment:
I added this because handleError uses a newline. Can you change it to pre-wrap instead of removing it?